jobboard/models.py

```diff
@@ -97,5 +102,12 @@ def get_jobs_by_company_name(cls, company_name):
         """
         return set(cls.objects.filter(company=Company.objects.get(name=company_name)))
 
-    def __str__(self):
+    @classmethod
+    def get_jobs_by_recruiter(cls, recruiter_name):
+        """
+        Allows students to search for a job based on the company
```

Comment:
I think that a better description will be based on the recruiter's name and maybe even pass the send the recruiter himself as an object and comparing with their id because can be two recruiter's with the same name. you can look at what I did in the job_application model.

Reply:
Yes. Agreed.
On it

Reply:
It can get a recruiter instance as an argument

Reply:
@yaelerdr ?

Reply:
Changed that to be a recruiter instance @guy9050